localstack/services/apigateway/integration.py

```diff
@@ -682,12 +698,17 @@ def invoke(self, invocation_context: ApiInvocationContext):
         if "Accept" not in headers:
             headers["Accept"] = "application/json"
 
-        template = integration.get("requestTemplates").get(APPLICATION_JSON)
-        if "GetQueueUrl" in template or "CreateQueue" in template:
+        if invocation_context.is_websocket_request():
+            template_key = self.render_template_selection_expression(invocation_context)
+            payload = self.request_templates.render(invocation_context, template_key)
```

Comment:
I think this should be the following, from what I've seen in our call yesterday:
```suggestion
            payload = self.render_template_selection_expression(invocation_context)
```

Reply:
no this is correct, the template_key is rendered and becomes the request template

Reply:
Ah my bad! I see it now! Thanks a lot 👍